src/behaviors/Column.js

```diff
@@ -228,7 +228,7 @@ Column.prototype = {
             }
         }
 
-        if (autoSized) {
+        if (fireResizeEvent && autoSized) {
```

Comment:
We now only fire events when the changes is made by user.

Reply:
everytime new column comes in with width, this will fire one event for each of those columns.
And it cause config out of sync issue in VC. And those event are not require at all